src/Main.java

```diff
@@ -0,0 +1,59 @@
+import java.util.function.BiFunction;
+import java.util.function.IntFunction;
+
+import static java.lang.Math.sqrt;
+
+public class Main {
+    public static void main(String[] args) {
+
+        //Task 1
+        System.out.println("Task1");
+        BiFunction<Integer, Integer, Integer> compare = (Integer _a, Integer _b) -> {
+            if (_a > _b) {
+                return _a;
+            } else return _b;
```

Comment:
It's better to use {} here (and in the other your code, including 'for' loops) to follow the Java code conventions